taxi/views.py

```diff
@@ -1,9 +1,13 @@
 from django.contrib.auth.decorators import login_required
+from django.http import HttpRequest, HttpResponse, HttpResponseRedirect
 from django.shortcuts import render
 from django.urls import reverse_lazy
 from django.views import generic
 from django.contrib.auth.mixins import LoginRequiredMixin
 
+from .forms import (DriverCreationForm,
+                    DriverLicenseUpdateForm,
+                    CarCreationForm)
```

Comment:
Place each import on a new line so that there would be no imports on the same line with opening and closing parenthesis. It will help to improve the code style and readability